src/lib/wallet/GoodWalletProvider.jsx

```diff
@@ -205,6 +208,11 @@ export const GoodWalletProvider = ({ children, disableLoginAndWatch = false }) =
 
         await wallet.ready
 
+        posthog?.register({
```

Comment:
dont think this is required, in anycase it is already in analytics init or should be there

Reply:
removed